utility/src/main/java/org/oppia/android/util/profile/ProfileNameValidator.kt

```diff
@@ -0,0 +1,55 @@
+package org.oppia.android.util.profile
+
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Utility to validate that profile names are correctly formatted. */
+@Singleton
```

Comment:
This can be omitted--scopes are only needed if there's state (otherwise it's fine for the Dagger provider to recreate the class).

Reply:
done